### ISSUE TITLE
Improve lore2sphinx buildbot results for meta docs in `projects/core/howto/`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4866]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4866|
|Type|enhancement|
|Created|2011-02-11 05:21:20Z|
|Branch|https://github.com/twisted/twisted/tree/metadocs-sphinx-4866|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for these documents:

  * debug-with-emacs
  * internet-overview
  * glossary
  * vision
  * design
  * upgrading

<details><summary>Searchable metadata</summary>

```
trac-id__4866 4866
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_metadocs_sphinx_4866 branches/metadocs-sphinx-4866
branch_author__screwtape__jdb screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297401680000000 1297401680000000
changetime__1300154017000000 1300154017000000
version__None None
owner__Screwtape Screwtape

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30664]) Branching to 'metadocs-sphinx-4866'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30665]) Merge the meta-ish doc-y parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch

refs [#4866](https://github.com/twisted/twisted/issues/4866)
refs #4568

**Comment by twisted-trac** [comment]
|[<img alt="tenth10th's avatar" src="https://avatars.githubusercontent.com/u/854235?s=50" width="50" height="50">](https://github.com/tenth10th)<a name="note_4"></a>|@tenth10th set owner to Screwtape|
|-|-|

**debug-with-emacs**: The capitalization changes are good! However, the Pokey The Penguin comic ([http://yellow5.com/pokey/archive/pokey411_3.gif]) from the Lore version of the page has been removed, while the footnote describing it is still in place. You should probably remove the footnote as well, or reinstate the Penguin. (I'm in favor of Pokey, myself.) 

**defer**: This wasn't listed in the documents for this ticket, but it needed some work, so who cares. ;-) All the corrections look good, but I noticed that at some point between Lore and Sphinx, in the first line of the "Other behaviours" section, we've lost the space between the colon and the opening code tag:
```
(...)modify its behaviour:<code>fireOnOneCallback</code>(...)
```

**internet-overview**: Looks good!

**glossary**: All good changes... The only thing I'd reconsider is the removal of the (currently broken) link from the "Spread" entry title: We could keep it as a link to core/howto/pb.html, which is titled "Overview Of Twisted Spread" and seems like a good place to send readers who want to learn more about Spread.

**vision**: Looks good!

**design**: Looks good!

**upgrading**: Looks good, but I noticed that under "Rebuild: Loading New Code Without Restarting", the API link is being removed from the first mention of Versioned (seemingly because it was broken?) But the second appearance of Versioned in that section has a working link to [http://twistedmatrix.com/documents/current/api/twisted.persisted.styles.Versioned.html]; It seems like the first mention should link to the same place. Later mentions of both rebuild and Versioned aren't links, but that seems to be on purpose. (I'm guessing the pattern is that the first mention in a paragraph is a link and the rest aren't?)

**Comment by twisted-trac** [comment]
|[<img alt="tenth10th's avatar" src="https://avatars.githubusercontent.com/u/854235?s=50" width="50" height="50">](https://github.com/tenth10th)<a name="note_5"></a>|@tenth10th commented|
|-|-|

Uh oh, these changes to the "defer" page are also (more properly) included in the **defer-sphinx-4868** branch... They should be reverted from this branch! I'll just copy my comments about the defer page to [#4868](https://github.com/twisted/twisted/issues/4868).

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_7"></a>|@khorn commented|
|-|-|

Remaining issues are not (strictly) conversion problems, so are now under [#4949](https://github.com/twisted/twisted/issues/4949).

Merge this.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_8"></a>|@khorn set status to `closed`|
|-|-|

(In [31099]) Merge metadocs-sphinx-4866

Author: Screwtape
Reviewers: tenth
Fixes: [#4866](https://github.com/twisted/twisted/issues/4866)

Improve buildbot results for following docs in lore to sphinx conversion.
